hail/python/test/hail/fs/test_worker_driver_fs.py

```diff
@@ -160,9 +161,13 @@ def test_can_access_public_blobs():
 
 @run_if_azure
 @fails_local_backend
-def test_qob_can_use_sas_tokens():
+async def test_qob_can_use_sas_tokens():
     vcf = resource('sample.vcf')
-    account = AzureAsyncFS.parse_url(vcf).account
+    fs = AzureAsyncFS()
+    try:
+        account = fs.parse_url(vcf).account
+    finally:
+        await fs.close()
```

Comment:
I think this change is no longer needed since `parse_url` is now static

Reply:
done